### PR TITLE
Documentation for arrow functions

### PR DESCRIPTION
**Pull request by ARUNBALACHOCKALINGAM**
<hr>

## Description 📜

there was no documentation for JavaScript arrow functions, so I have created this issue and added this documentation which explains the syntax and various use cases of JavaScript arrow function
Fixes #8731

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [X] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [X] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/Javascript/Arrow Functions/Readme.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Arrow Functions in javascript
+<hr>
+##What are arrow functions?
```
Comment (ghulamyazdani) [inline Web_Development/Javascript/Arrow Functions/Readme.md:3]: Give space between # and the content to get the results

File: Web_Development/Javascript/Arrow Functions/Readme.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Arrow functions are introduced to write compact functions in a abbreviated way
```
Comment (ghulamyazdani) [inline Web_Development/Javascript/Arrow Functions/Readme.md:5]: elaborate more about arrow function.

**Review by ghulamyazdani — CHANGES_REQUESTED** [review]

File: Web_Development/Javascript/Arrow Functions/Readme.md (hunk 1 above)
Comment (ARUNBALACHOCKALINGAM) [reply Web_Development/Javascript/Arrow Functions/Readme.md]: Made the changes, thanks for the feedback

File: Web_Development/Javascript/Arrow Functions/Readme.md (hunk 2 above)
Comment (ARUNBALACHOCKALINGAM) [reply Web_Development/Javascript/Arrow Functions/Readme.md]: Made the required changes

**Comment by ghulamyazdani** [comment]
There is still the same issue please fix it.
It should be bold text like other heading texts
image

**Review by ghulamyazdani — COMMENTED** [review]
LGTM 🥳 Thanks for your valuable contribution

**Review by ghulamyazdani — APPROVED** [review]

**Pull request merged**